backend/app/models/project_developer.rb

```diff
@@ -7,11 +7,12 @@ class ProjectDeveloper < ApplicationRecord
   has_many :favourite_project_developers, dependent: :destroy
   has_many :project_involvements, dependent: :destroy
   has_many :involved_projects, through: :project_involvements, source: :project, dependent: :destroy
+  has_many :project_developer_priority_landscapes, dependent: :destroy
+  has_many :priority_landscapes, through: :project_developer_priority_landscapes, source: :priority_landscape, dependent: :destroy
 
   validates :categories, array_inclusion: {in: Category::TYPES}, presence: true
   validates :impacts, array_inclusion: {in: Impact::TYPES}
   validates :project_developer_type, inclusion: {in: ProjectDeveloperType::TYPES, allow_blank: true}, presence: true
-  validates :mosaics, array_inclusion: {in: Mosaic::TYPES}
```

Comment:
I see `mosaics` collection was not required, and running migration on re-seeded database does not work.

Reply:
Ouch, I thought that it will be empty array in such case, hmm, will fix migration ;)